### ISSUE TITLE
Generate the tmrca of every node

### ISSUE THREAD

**Issue by Dhihram** [emvolz/treedater#23]
I have utilized  `parboot` to find tmrca and HPD 05% of my tree. Can `parboot`  also extract the interval of every node in my tree?

**Comment by emvolz** [comment]
Yes, it is possible. The output of parboot should contain a list called $trees. 
Each element of $trees is itself a treedater fit. If you are interested in a particular node (e.g. 'x' ), iterate over these trees, find the distance from the root (ape::node.depth.edgelength) and add to the estimated tmrca of the total tree. 
Example: 
```
sapply( pb$trees, function(tr) tr$timeOfMRCA + node.depth.edgelength(tr)[x] )  |> quantile(prob=c(.025,.975))
```

**Comment by Dhihram** [comment]
Thank you very much.

**Comment by Dhihram** [comment]
Btw, Is it right to say the quantile as HPD? Or is it CI?

**Comment by emvolz** [comment]
I would call it a CI

**Comment by Dhihram** [comment]
Thank you very much
